users/models.py

```diff
@@ -39,6 +39,7 @@ class CustomUser(AbstractBaseUser):
     is_doctor    = models.BooleanField(default=False, null=True)
     date_joined  = models.DateTimeField(auto_now_add=True)
     is_superuser = models.BooleanField(default=False)
+    is_active    = models.BooleanField(default=True)
```

Comment:
why are you adding `is_active`?

Reply:
I'm adding it so the user will be unable to login until they activate their account. In views.py, I have it so when the user is created, its is_active value is set to false, then it is set to true when they click the activation link in their email.